README.md

```diff
@@ -18,6 +18,9 @@ Please note that the action executes Az CLI script in a docker container. This m
   1. If script sets up an environment variable, it will not take effect in host and hence subsequent actions shouldn't rely on such environment variable.
   2. There is some restriction on how cross action file read/write is done. GITHUB_WORKSPACE directory in host is mapped to working directory inside container. So, if the action wants to create a file, which will be read by subsequent actions, it should do so within current working directory tree.
 
+> **Warning**

```

Comment:
It is interesting that GitHub-flavored markdown supports Warning:

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

According to https://github.com/orgs/community/discussions/16925, it seems `[!WARNING]` is the recommended format. It is also aligned with our Microsoft docs repos: https://github.com/MicrosoftDocs/azure-docs-cli/blob/b174d42930d6f69fd07dd589eaff4a545a57d316/docs-ref-conceptual/install-azure-cli-windows.md?plain=1#L29

```
> [!IMPORTANT]
> After the installation is complete, you will need to **close and reopen any active terminal window to use the Azure CLI**.
```